.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Could you remove `.vscode` folder from commit?

Reply:
yes!